tools/rna_tools/intarna/intarna.xml

```diff
@@ -52,26 +52,26 @@
                 label="Disable seed constraint entirely for computation" help=""/>
             <param argument="--seedBP" type="integer" value="7" min="2" max="20"  label="Min. number of basepairs in seed" help="number of inter-molecular base pairs within the seed region"/>
             <param argument="--seedMaxUP" type="integer" value="0" min="0" max="20"
-                label="Max. overall number of unpaired bases (query+target)" help=""/>
+                label="Max. overall number of unpaired bases in seed (query+target)" help=""/>
         </macro>
         <macro name="interaction_macro">
             <param argument="--mode" type="select" label="Prediction mode" help="">
-                <option value="H" selected="true">Heuristic (fast and low memory)</option>
-                <option value="M">Exact and low memory</option>
-                <option value="E">Exact (high memory)</option>
+                <option value="H" selected="true">Heuristic ('H')</option>
+                <option value="M">Exact but slow ('M')</option>
+                <!-- <option value="E">Exact (high memory)</option> -->
```

Comment:
Why removed? Memory should not be a problem?

Reply:
no output difference.. mode only existent for testing purpose